docs/source/build_instructions.md

```diff
@@ -142,7 +142,7 @@ $ cd src
 #### Linux and Mac
 
 Shaka Packager uses [Ninja](https://ninja-build.org) as its main build tool,
-which is bundled in depot_tools.
+which is bundled in `depot_tools.
```

Comment:
do you mean `depot_tools` here?

Reply:
you are right, `depot_tools` is the way to go (my minor proposal)